### ISSUE TITLE
For kind:"workshop" the logic is missing in navbar.html

### ISSUE THREAD

**Issue by sk-sahu** [carpentries/carpentries-theme#6]
We (me, @ErinBecker and @alanaschick) were trying to modify `kind:"workshop"` in [genomics-workshop](https://github.com/datacarpentry/genomics-workshop/blob/2ed2c048134e7b02835a72924b4ed1c0299c9920/_config.yml#L21) which is currently - 

```config
# What kind of thing is this ("workshop" or "lesson")?
kind: "lesson"
```

And found out the [logic is only implemented for `kind:"lesson"` in navbar.html](https://github.com/carpentries/carpentries-theme/blob/5cbd3a631a379af8947b2ca339f234c861c34055/_includes/navbar.html#L51)

**Comment by fmichonneau** [comment]
The variable `kind: "workshop"` is used to set up workshop websites. In other words, this site (and the ones instructors and hosts create using this template): https://carpentries.github.io/workshop-template/ and not a workshop overview page like the genomics workshop page. A workshop overview page is also created by using `kind: "lesson"`. Because people often need to customize the content of their workshops, I think that using remote theme in these cases would make things more confusing. But that's not the topic for this issue.

What are you trying to modify from the genomics workshop overview site?

**Comment by sk-sahu** [comment]
Yes we trying to remove the **Episodes** tab (as it doesnt servers any purpose in a workshop site.)

![image](https://user-images.githubusercontent.com/23085664/89908044-0e3f4580-dc0b-11ea-80cb-8dfd1d889cce.png)

**Comment by fmichonneau** [comment]
Thanks! The current template doesn't support this, but I opened a PR to address this issue: https://github.com/carpentries/styles/pull/491

**Comment by fmichonneau** [comment]
The PR is merged, and the remote theme has been updated. The Episodes dropdown should be hidden at the next build of the lesson.
